docs/src/configuring_zed.md

```diff
@@ -956,14 +956,70 @@ See Buffer Font Features
 
 ## Theme
 
-- Description: The name of the Zed theme to use for the UI.
+- Description: The name and mode of the Zed theme to use for the UI.
 - Setting: `theme`
```

Comment:
The `theme` setting can be _either_ a name of a theme or an object containing the `mode`, `dark`, and `light` themes.

We'll want to convey in the docs that both forms are supported.

Reply:
@maxdeviant I made some changes to convey that both forms are supported. Let me know what you think.